src/Raven.Server/Documents/Sharding/Handlers/Processors/Batches/ShardedBatchHandlerProcessorForBulkDocs.cs

```diff
@@ -55,6 +58,24 @@ protected override ValueTask WaitForReplicationAsync(TransactionOperationContext
         return ValueTask.CompletedTask;
     }
 
+    private ShardedBatchBehavior GetBatchBehavior()
```

Comment:
Okay, so for < 6.0, we are defaulting to multi bucket, but for >= 6.0 we are defaulting to single bucket?

Isn't that likely to be really annoying for the default use case?

I *want* that to be the default, by the way, but it is going to be something that the user needs to subscribe to explicitly, or they will fail from the very first steps.